spec/unit/log_transitions_spec.rb

```diff
@@ -6,7 +6,7 @@
   after  { FiniteMachine.logger = ::Logger.new($stderr) }
 
   it "logs transitions" do
-    fsm = FiniteMachine.new log_transitions: true do
+    fsm = FiniteMachine.new name: "TrafficLights", log_transitions: true do
```

Comment:
We need a test case when the name is not specified. I'd probably keep this test as it was without the name and test the short UUID generation. Let me know if you need any pointers on how to test this.

Reply:
Understood. I'll give it a shot and reach out if I run into trouble. Thanks.

Reply:
Hi @piotrmurach: Just wanted to chime in and let you know that I got distracted but will get back to this in the next couple of days. Thanks.